xpdacq/calib.py

```diff
@@ -122,35 +122,103 @@ def run_calibration(exposure=60, dark_sub=True, calibrant_file=None,
                      wavelength=wavelength, detector=detector,
                      gaussian=gaussian)
 
-    # masking
-    print('INFO: create mask')
-    mask = get_mask(img, ai, glbl.mask_dict, save_name=None)
+    # save glbl attribute for xpdAcq
+    timestr = _timestampstr(time.time())
+    basename = '_'.join(['pyFAI_calib', calibrant_name, timestr])
+    glbl.calib_config_dict = ai.getPyFAI()
+    Fit2D_dict = ai.getFit2D()
+    glbl.calib_config_dict.update(Fit2D_dict)
+    glbl.calib_config_dict.update({'file_name':basename})
+    glbl.calib_config_dict.update({'time':timestr})
+    # FIXME: need a solution for selecting desired calibration image
+    # based on calibration_collection_uid later
+    glbl.calib_config_dict.update({'calibration_collection_uid':
+                                   calib_collection_uid})
+    # save yaml dict used for xpdAcq
+    yaml_name = glbl.calib_config_name
+    with open(os.path.join(glbl.config_base, yaml_name), 'w') as f:
+        yaml.dump(glbl.calib_config_dict, f)
 
     return ai
 
 
-def get_mask(img, geometry_object, mask_dict, save_name=None):
+def run_mask_builder(exposure=60, dark_sub=True,
```

Comment:
check with max what is an appropriate default exposure for empty kapton.  it might be much longer than this.